### PR TITLE
Add spellcheck for docs

### PR DESCRIPTION
**Pull request by davidfaulkner12**
While I was blocked on onboarding I went through and did a quick spellcheck Github Action for our docs. There were quite a few random misspellings (as would inevitably happen without CI). Not a perfect solution, but at least our public docs won't have any more tipos like "Fennnel" :-)

### CODE REVIEW

**Comment by davidfaulkner12** [comment]
An example of failing spellcheck: https://github.com/fennel-ai/client/actions/runs/5932533994/job/16086498008?pr=272

**Comment by aditya-nambiar** [comment]
This is awesome!

**Review by aditya-nambiar — APPROVED** [review]

File: .wordlist.txt [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+scalable
+schemas
+signup
+snowflakecomputing
+stateful
+str
+strftime
+struct
+tiering
+uid
```
Comment (aditya-nambiar) [inline .wordlist.txt:178]: Can we keep it sorted across cases ?

File: .wordlist.txt [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+protobuf
+protobufs
+quickstart
+realtime
+regex
+regexes
+repo
+runtime
+scalability
+scalable
```
Comment (aditya-nambiar) [inline .wordlist.txt:169]: Can we add extractor, depends_on, version, inputs, ( all our sources maybe ? kinesis, bigquery etc )

Comment (davidfaulkner12) [reply .wordlist.txt:142]: Most of those are already part of the dictionary -- I'm not quite sure why `scalable` and `scalability` aren't. It's just this text file (which doesn't even need to be sorted) so trivial to add things later.

File: .wordlist.txt (hunk 1 above)
Comment (davidfaulkner12) [reply .wordlist.txt:165]: Done.

**Pull request merged**